src/fdk_rdf_parser/fdk_rdf_parser.py

```diff
@@ -18,6 +19,20 @@ def isTypeDataset(graph: Graph, topic: URIRef) -> bool:
     return False
 
 
+def parseDataServices(dataServiceRDF: str) -> Dict[str, DataService]:
+    dataServices: Dict[str, DataService] = {}
+
+    dataServicesGraph = Graph().parse(data=dataServiceRDF, format='turtle')
+
+    for dataServiceURI in dataServicesGraph.subjects(
+            predicate=RDF.type, object=dcatURI("DataService")
```

Comment:
use CatalogRecord instead of DataService, then you'll have metadata available

Reply:
and you then need to copy the 'isTypeDataset' functionality

Reply:
Do you mean these metadata?

![image](https://user-images.githubusercontent.com/3957655/85375576-cd924c80-b536-11ea-94d4-1800256c7348.png)

Reply:
And should I duplicate them to every data service?

Reply:
duplicate?
They all have their own individual CatalogRecord

Reply:
you might be using an old version of dataservice-harvester, recently added that metadata for DataService is included when you get a Catalog